plugins/meta/claude-docs/tests/test_firecrawl_mcp_docs.py

```
@@ -0,0 +1,76 @@
+        assert success is True
+        assert "Claude Code Documentation" in content
+
+
+@pytest.mark.asyncio
+async def test_error_handling_robustness():
+    """Test that Firecrawl handles errors gracefully."""
+    from firecrawl_mcp_docs import download_page_firecrawl
+
+    # Should handle failures without crashing
+    url = "https://invalid-url-that-will-fail.com"
+    success, content, metadata = await download_page_firecrawl(url, Path("/tmp"))
```

Ruff (0.14.3)

Unpacked variable `content` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)